src/components/Editor.jsx

```diff
@@ -114,6 +116,14 @@ class Editor extends React.Component {
     }
   }
 
+  _toggleEditorTextSize(editorTextSizeIsLarge) {
+    if (editorTextSizeIsLarge === true) {
+      this._editor.setFontSize('18px');
```

Comment:
Probably best to extract these into `const`s at the top of the file (e.g. `normalFontSize` and `largeFontSize`)